### ISSUE TITLE
Suggestions for 0.8.0

### ISSUE THREAD

**Issue by jbrodriguez** [wkh237/react-native-fetch-blob#68]
Hi @wkh237, I think it's great how 0.8.0 is coming along.

I'd like to make two suggestions:
- I see you're adding a timeout option. Can it be set in milliseconds rather than whole seconds ?
- Is it possible to get back the http status codes in the response. For example, if you request a resource that doesn't exist, send to the js context the 404. Right now, the status code is hidden.

In any case, it's an awesome work, I'm looking forward to use it when it's ready

**Comment by wkh237** [comment]
@jbrodriguez , thank you very much for your suggestions. First, I think it's good to change timeout unit to millisecond rather than second, I'll change the design.  

As for the status code supporting, that has been implemented but I'm not yet write it to the document. In `0.8.0` the response object will have an extra method call `info()`, it will return an object like this :

```js
type RNFetchBlobResponseInfo = {
  taskId : string,
  state : number,
  headers : any,
  status : number,
  respType : 'text' | 'blob' | '' | 'json'
  rnfbEncode : 'path' | 'base64' | 'ascii' | 'utf8'
}
```

I've tested this with XMLHttpRequest polyfill, check [the test case](https://github.com/wkh237/react-native-fetch-blob/blob/0.8.0/test/test-xmlhttp.js#L90) if you'd like to know more detail 😃
